src/wallet/states/responding.ts

```diff
@@ -6,49 +6,73 @@ export const RESPONDING = 'RESPONDING';
 // state types
 export const ACKNOWLEDGE_CHALLENGE = 'ACKNOWLEDGE_CHALLENGE';
 export const CHOOSE_RESPONSE = 'CHOOSE_RESPONSE';
-export const ACKNOWLEDGE_MUST_TAKE_MOVE = 'ACKNOWELDGE_MUST_TAKE_MOVE';
+export const SELECT_CHALLENGE_RESPONSE = 'SELECT_CHALLENGE_RESPONSE';
```

Comment:
I see you now have `SELECT_CHALLENGE_RESPONSE` and `CHOOSE_RESPONSE`. It isn't clear to me what the difference between the two of these are. Do we definitely need both of them?

Reply:
Hmm, the naming should probably be clarified. `SELECT_CHALLENGE_RESPONSE` should probably be `SELECT_RESPOND_WITH_MOVE`

Reply:
Presumably you the move you pick would be part of the `CHOOSE_RESPONSE` e.g.
```ts
enum Response { RespondWithMove, Refute, RespondWithAlternativeMove }

chooseResponse(response: Response) { ... }
```

Reply:
@tomclose Right, sorry I think I was thinking of actions. I just realized that `SELECT_CHALLENGE_RESPONSE` is actually not used, so I must of introduced it and forgot about it. :)